frontend/public/components/utils/tile-view-page.jsx

```diff
@@ -227,6 +235,17 @@ const filterByGroup = (items, filters) => {
   );
 };
 
+export const filterByGroupByDropdown = (items, groupBy) => {
+  if (groupBy === groupByTypes.Operator) {
+    const installedOperators = _.filter(items, (item) => item.kind === 'InstalledOperator');
+    const nonOperators = _.filter(items, (item) => item.kind !== 'InstalledOperator');
+    const groupedOperators = _.groupBy(installedOperators, (item) => item.obj.csv.metadata.name);
+    const groupAllItems = { ...groupedOperators, 'Non-Operators': nonOperators };
```

Comment:
Why add a hyphen in between when its a string?
```suggestion
    const groupAllItems = { ...groupedOperators, 'Non Operators': nonOperators };
```